packages/stark-ui/src/modules/date-range-picker/components/date-range-picker.component.ts

```diff
@@ -117,67 +191,140 @@ export class StarkDateRangePickerComponent extends AbstractStarkUiComponent impl
 	public dateRangeChanged: EventEmitter<StarkDateRangePickerEvent> = new EventEmitter<StarkDateRangePickerEvent>();
 
 	/**
-	 * Reference to the start datepicker embedded in this component
+	 * Filter function or a string
+	 * Will be applied to both date-picker
 	 */
-	@ViewChild("startPicker")
-	public startPicker!: StarkDatePickerComponent;
+	@Input()
+	public dateFilter?: StarkDatePickerFilter;
 
 	/**
-	 * Reference to the end datepicker embedded in this component
+	 * Timestamp Mask Configuration to apply on the start/end date-picker.
+	 * If `true` is passed, the default mask config is applied: {DEFAULT_DATE_MASK_CONFIG|DEFAULT_DATE_MASK_CONFIG}
+	 * If `false` is passed or if `dateMask` is not present, the directive is disabled.
+	 * If a `StarkTimestampMaskConfig` is passed, it is set as the date mask config.
 	 */
-	@ViewChild("endPicker")
-	public endPicker!: StarkDatePickerComponent;
+	@Input()
+	public dateMask?: StarkDatePickerMaskConfig;
+
+	/**
+	 * Whether the datepickers are disabled
+	 */
+	@Input()
+	public set disabled(val: boolean) {
+		if (val) {
```

Comment:
I think we should log a warning here in case the developer is using the `formGroup` input and also this `disabled` input... which might cause some confusion and prone to errors on his side. So we could warn him about this so if he uses the `formGroup`, he can simply disable the group. What do you think?

Reply:
It's weird, but normally you can use both. I added a TMP commit to demonstrate

Reply:
Yes, exactly. It is a weird scenario but yet is possible. So we should at least warn the developer that he is providing 2 inputs which are not intended to be used together...